lib/index.d.ts

```diff
@@ -0,0 +1,228 @@
+/// <reference types="node" />
+
+export interface RenderOptions {
+  page?: number
+  density?: number
+  quality?: number | undefined
+  outline?: boolean
+}
+
+export interface SaveOptions extends RenderOptions {
+  format?: string | undefined
+  matte?: string
+}
+
+export class Canvas {
+  /** @internal */
+  static contexts: WeakMap<Canvas, readonly CanvasRenderingContext2D[]>
```

Comment:
Marked as `internal` because it is used in tests, but not documented in the README